Change setup.cfg so that the bundled zlib and bzip2 are not used.  This cannot
currently be achieved using "--library z,bz2" as instructed in the setup.py.

diff --git a/setup.cfg b/setup.cfg
index c054092..080992e 100644
--- a/setup.cfg
+++ b/setup.cfg
@@ -1,7 +1,7 @@
 [build_ext]
 define = SEQAN_HAS_BZIP2,SEQAN_HAS_ZLIB
 undef = NO_UNIQUE_RC
-# libraries = z,bz2
+libraries = z,bz2
 ## if using system libraries
 include-dirs = lib:third-party/zlib:third-party/bzip2:third-party/seqan/core/include:third-party/smhasher
 # include-dirs = lib
